cli/cmd/replicaset.go

```diff
@@ -813,19 +813,25 @@ func getOrchestrator() (replicaset.Orchestrator, error) {
 	return orchestrator, nil
 }
 
-// internalReplicasetRebootstrapModule is a "rebootstrap" command for the replicaset module.
-func internalReplicasetRebootstrapModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
+// replicasetRebootstrapValidateArgs validates "rebootstrap" command arguments.
+func replicasetRebootstrapValidateArgs(cmd *cobra.Command, args []string) error {
 	if len(args) > 1 {
-		return util.NewArgError("only one instance supported for re-bootstrap")
+		return errors.New("only one instance supported for re-bootstrap")
 	}
 	if len(args) < 1 {
-		return util.NewArgError("instance for rebootstrap is not specified")
+		return errors.New("instance for rebootstrap is not specified")
+	}
+	if !strings.Contains(args[0], string(running.InstanceDelimiter)) {
+		return errors.New("an instance name is not specified. Please use app:instance format")
 	}
+	return nil
+}
 
+// internalReplicasetRebootstrapModule is a "rebootstrap" command for the replicaset module.
+func internalReplicasetRebootstrapModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 	appName, instName, found := strings.Cut(args[0], string(running.InstanceDelimiter))
 	if !found {
-		return util.NewArgError(
-			"an instance name is not specified. Please use app:instance format.")
+		panic("Instance delimiter is missing unexpectedly.")
```

Comment:
```suggestion
		return util.NewArgError("Instance delimiter is missing unexpectedly.")
```
Why should we trigger panic here?

Reply:
It's a kind of assert. Function that validate arguments must make this `panic` unreachable.

Reply:
I've removed this `panic` just to keep consistent code (there are no such "assertions" in other places).